### ISSUE TITLE
Dicontinued?

### ISSUE THREAD

**Issue by marler8997** [evilrat666/directx-d#8]
After opening my issue, I saw there was a `[DISCONTINUED]` label in the repo description.  Why was it discontinued?  Do you have a recommendation for another library or did you just lose interest in the project?

**Comment by evilrat666** [comment]
In short: 
I no longer "need" it. It is there to not break people's projects. I put it in "passive maintenance" mode, i.e. it is unlikely that I fix something myself, but still accept pull requests and clean up if needed.

Cool story:
I was busy at the moment, so I asked for someone who can step in on the forums, there was zero interest which is kind of frustrating considering that D still probably the only language out there that has DirectX bindings at all.

Anyway, people suggested to not rush to delete the package from dub, so I keep it there to not break existing people's code due to missing dependencies. 

I have no idea how much people still using it, but I do read GH notifications on activity here.

Alternatives:
There was something like Aurora project but they had very sparse bindings. Another option is to use IDL to D converter, I think webfreak (code-d author) has one, or normal binding generators like dpp/dstep/gentool.

**Comment by marler8997** [comment]
Yeah I tried the aurora one and it's definitely missing stuff.  Well good to know where the project is at.  I see there are a fair number of forks.  I'll think about if I should fork it myself or use someone else's or think of something else.

**Comment by johnburton** [comment]
I forked this and made a local copy when I saw it was discontinued as insurance against it going away as I occasionally play with it. I presume that's within the spirit of the license :) When you originally said this was discontinued I thought about volunteering but neither D or DirectX are my main skills, and I've only really used this in passing for playing about with some graphics so I'd be a very poor maintainer.

I do keep coming back to this though, and I'm pleased it's still here and wanted to thank you for making and continuing to maintain this on whatever level you can. It's a very good library and I'd hate us to lose it. If the alternative was it going away I guess I would want to keep a fork going until a more suitable maintainer could be found.
